test/functional/plugin/pack_spec.lua

```diff
@@ -1136,6 +1136,7 @@ describe('vim.pack', function()
 
       it('has in-process LSP features', function()
         t.skip(not is_jit(), "Non LuaJIT reports errors differently due to 'coxpcall'")
+        track_nvim_echo()
```

Comment:
nit: is "spy" the usual name for this?
```suggestion
        spy_nvim_echo()
```

Reply:
Honestly first time hearing it :eyes: